app/models/assignment.rb

```diff
@@ -619,6 +619,10 @@ def questionnaire_ids(round)
     questionnaire_ids
   end
 
+  def is_pair_programming_enabled?(assignment_id)
```

Comment:
generally don't use predicates when writing a method like this. also what is the utility of this parameter. wouldn't you be calling this on the assignment object in question